lib/ex_admin/themes/admin_lte2/filter.ex

```diff
@@ -121,7 +122,12 @@ defmodule ExAdmin.Theme.AdminLte2.Filter do
     id = "q_#{owner_key}"
     name_label = field_label(name, defn)
     repo = Application.get_env :ex_admin, :repo
-    resources = repo.all assoc
+    name_field = ExAdmin.Helpers.get_name_field(defn.resource_model)
```

Comment:
it's `assoc` and not `defn.resource_model`, the child table might not have that field name.